### ISSUE TITLE
CSS Transitions not animating

### ISSUE THREAD

**Issue by GodGardensGuns** [brave/brave-ios#5177]
### Description: CSS Transitions are not animating properly. It's as though the CSS value goes from one state to another with no transition applied.

I first noticed it with transitions that are supposed to be triggered by IntersectionObserver. That API may have something to do with it.

I'm not sure if the issue extends to CSS Animations.

### Steps to Reproduce 
  1. Open a website that uses CSS Transitions. [https://anytype.io](https://anytype.io) and [https://vercel.com/workflow](https://vercel.com/workflow) are good examples.
2. The transitions don't work.

**Actual result:**

https://user-images.githubusercontent.com/58375683/160860992-a0dc9433-9725-463f-a4b6-a364ec243251.MOV

https://user-images.githubusercontent.com/58375683/160861099-37f4cb02-f586-48dd-846e-18280d8ab1a7.MOV

**Expected result:**
(these were recorded with the Chrome browser, but look the same as when using Safari)

https://user-images.githubusercontent.com/58375683/160862069-c31611ee-44f8-4dd9-af04-594341f8814f.MOV

https://user-images.githubusercontent.com/58375683/160862164-1ca0b43c-0cc7-49a8-a1c5-20fee0f4b298.MOV

**Reproduces how often:**

Easily Reproduced

**Brave Version:**

Version 1.36 (22.3.11.16) BraveCore 1.36.108 (99.0.4844.51)

**Device details:** 

iPhone 13
iOS 15.2.1

**Website problems only:**
- did you check with Brave Shields down?

Yes.

- did you check in Safari/Firefox (WkWebView-based browsers)? 

Yes (Safari).

**Comment by Brandon-T** [comment]
Odd bug. I checked those exact sites in both Safari and Brave and wasn't able to reproduce the issue with and without shields.
Which shields do you have turned on?

https://user-images.githubusercontent.com/1530031/160909129-dfcf9a08-d2e6-4d61-b3e2-025c21c6cec6.MP4

**Comment by Brandon-T** [comment]
Hi, I have tried to reproduce the issue with all shields, and with no shields, and I am still unable to reproduce it.
What shields do you have on when the issue occurs?
Does turning off shields help?

https://user-images.githubusercontent.com/1530031/160910919-ceb4c21f-a973-424f-9c7e-f8e67d8bd0b9.MP4

**Comment by GodGardensGuns** [comment]
I'm using SwiftKey as my device keyboard. Oddly enough, when I turned SwiftKey off and turned the default iOS keyboard back on, transitions started working again. I reverted to using SwiftKey and turned off the default keyboard and transitions are still working properly.

Before I changed my keyboards, transitions randomly started working again, but they were still acting off:

https://user-images.githubusercontent.com/58375683/160920097-fbe03041-d7b4-4682-86e8-53aa16c93ef0.MOV

Here are the shields I've been using through this whole process:

![image](https://user-images.githubusercontent.com/58375683/160920174-700b3b8a-a1bd-4835-b3a8-6502d36e345f.png)

They're the default ones as far as I'm aware (I've never changed them).

Here is how transitions look now:

https://user-images.githubusercontent.com/58375683/160920304-5b5046c4-9ce7-4be4-8399-10120e6dc7ff.MOV

**Comment by GodGardensGuns** [comment]
This finally seems fixed after a recent update.

**Comment by rdwallis** [comment]
I ran into this problem on Version 1.47 (23.2.1.14)
BraveCore 1.47.189 (109.0.5414.119)
Device iPhone13,1 (iOS 16.3.1)

There was a multisecond delay in animations starting and then the transition caught up a few seconds later.  Which meant that short animations didn't play at all and just jumped to the end state once the delay ended.

The problem didn't occur in Safari but **I think I fixed it by resetting Safari Experimental Features**.
Navigate to Settings –> Safari –> Advanced –> Experimental Features, and then scroll to the bottom and tap "Reset All to Defaults."

Might have been a coincidence but the animations worked immediately after that.
